plaidml/op/lib/ops.cc

```diff
@@ -2140,6 +2142,14 @@ Value nms(const Value& value) {
   std::vector<TensorDim> Scatter_dims = {num_batches, num_classes, one};
   std::vector<TensorIndex> Scatter_idxs(3);
   Tensor Zero_scatter = edsl::Contraction(Scatter_dims, Scatter_idxs).assign(Zero(0));
+
+  std::function<edsl::Tensor(Tensor, Tensor)> IOU_compare;
+  if (hard_suppression) {
```

Comment:
Does the name `hard_suppression` mean the difference between `>` and `>=`? Isn't it the opposite of `soft-nms` or something? I'm not sure I understand in the correct way.

Reply:
Yes, I took this name from @XinWangIntel 's [comment](https://github.com/Flex-plaidml-team/plaidml/pull/102/files#diff-ad5ef91a2afc0f1c0b2ed9214ff2a6df00f49c2049c6eb8d319eac781c985825R2176), not sure if it can explain this functionality well.

Reply:
Yes, mainly for the equal case, only the nms reference shows this request